Pricing FAQs

kennethsequeira opened the pull request:
![image](https://user-images.githubusercontent.com/33246109/56111627-43d01200-5f76-11e9-9100-f40e3a45ccdf.png)

rmehta commented on erpnext_com/www/pricing/faq.md:
```diff
@@ -0,0 +1,67 @@
+##### Q. How is the fees calculated for the number of users?
+You only pay for the number of System users in your accounts. i.e those who can access the desk.
+
+##### Q. Who is a system user?
```
lets explain this without using "Desk". We can say, system user is someone who uses core modules - not portals. Customers, Suppliers, Students, Guardians etc are not system users.

Desk is going to change in v12 (pretty soon)

rmehta commented on erpnext_com/www/pricing/faq.md:
```diff
@@ -0,0 +1,67 @@
+We use Paypal as our payment gateway to handle international subscriptions. You do not need to have a Paypal account for continuing with the subscription.
+
+##### Q. I am running out of storage space/need to add more users. How do I go about doing so?
+You can add more space/users from within your account. Just go to **My Account > Upgrade > Purchase Add-ons**. 
```
show an animated GIF here

rmehta commented:
Please update with new pricing screenshot and merge!

kennethsequeira commented:
- Rewrote the page in html (based on the partner FAQ page)
- Removed references of desk as requested
- Added gifs for upgrades and subscription

![image](https://user-images.githubusercontent.com/33246109/59491041-be98a900-8ea3-11e9-84ce-e66c2ede31b4.png)

rmehta commented:
@kennethsequeira payment page is redesigned!

kennethsequeira commented:
@rmehta my bad. Took the screenshot with the old one. Here's the page with the redesigned payments page:

![image](https://user-images.githubusercontent.com/33246109/59582371-b6d04300-90f5-11e9-90eb-073de8a52143.png)

Here are the gifs added:

https://github.com/frappe/erpnext_com/blob/27b4af5f439dbd974b7d54a5cae7da2338a56d11/erpnext_com/public/images/subscribe.gif

https://github.com/frappe/erpnext_com/blob/27b4af5f439dbd974b7d54a5cae7da2338a56d11/erpnext_com/public/images/upgrade.gif

prasadarr commented on erpnext_com/www/pricing/faq.html:
```diff
@@ -0,0 +1,77 @@
+				<h5>How to proceed with the subscription payment?</h5>
+                You can pay for your ERPNext account from the application itself! Go to <b>My Account > Upgrade</b>.
+                <div> <!--class="col-sm-12 text-center mar-t-40"> -->
+                    <img src="/assets/erpnext_com/images/subscribe.gif" class="img-responsive"
```
the GIFs here represent v11, can a gif on v12 be made?

kennethsequeira replied:
There is no cloud account on v12 so far. Hence can't make a gif of the same

prasadarr commented on erpnext_com/www/pricing/faq.html:
```diff
@@ -0,0 +1,77 @@
+                You can pay for your subscription via Paypal or Credit card. If not, there's always Wire Transfer! Details for the same can be found <a href="https://erpnext.com/pricing/payment">here</a>                
+			</p>
+			<p class="mt-5">
+				<h5>Do I need to have a Paypal Account for subscription?</h5>
```
not done everywhere
